esmerald/responses/encoders.py

```diff
@@ -17,15 +20,48 @@ class ORJSONResponse(BaseJSONResponse):
     In the same way the JSONResponse is used, so is the `ORJSONResponse`.
     """
 
+    @staticmethod
+    def transform(value: Any, encoders: Any = None) -> dict[str, Any]:
+        """
+        The transformation of the data being returned.
+
+        Supports all the default encoders from Lilya and custom from Esmerald.
+        """
+        return cast(
+            dict[str, Any],
+            json_encode(
+                value,
+                json_encode_fn=orjson.dumps,
+                post_transform_fn=orjson.loads,
+                with_encoders=encoders,
+            ),
+        )
+
     def make_response(self, content: Any) -> bytes:
-        return orjson.dumps(
-            content,
-            default=self.transform,
-            option=orjson.OPT_SERIALIZE_NUMPY | orjson.OPT_OMIT_MICROSECONDS,
+        encoders = (
+            (
+                (
+                    *(encoder() if isclass(encoder) else encoder for encoder in self.encoders),
+                    *ENCODER_TYPES_CTX.get(),
+                )
+            )
+            if self.encoders
+            else None
+        )
+        return cast(
+            bytes,
+            json_encode(
+                content,
+                json_encode_fn=partial(
+                    orjson.dumps, option=orjson.OPT_SERIALIZE_NUMPY | orjson.OPT_OMIT_MICROSECONDS
```

Comment:
This is ok to override since you want orjson here but the transform should be at the base as it was before.